lms/static/scripts/frontend_apps/components/test/LMSFilePicker-test.js

```diff
@@ -301,6 +396,36 @@ describe('LMSFilePicker', () => {
     assert.calledWith(onSelectFile, file);
   });
 
+  it('does not invoke `onSelectFile` if chosen file is empty', async () => {
+    const onSelectFile = sinon.stub();
+    const wrapper = renderFilePicker({ onSelectFile });
+    await fakeApiCall;
```

Comment:
There are still `await fakeApiCall` calls here that should be replaced with `waitFor*` calls to wait until the file list is ready.